### ISSUE TITLE
Compass Sass not compiling

### ISSUE THREAD

**Issue by ipansensoy** [SuperIRis/basic-starter#1]
Sass Compass is not compiling

**Comment by SuperIRis** [comment]
You're right, the gulp-compass had a bug (compass itself works, but the integration with the plugin fails because of file structure). I'm following the issue and I'll update this when it is fixed. For the meantime, no Compass for the starter :(
